src/ClientNet462/Program.cs

```diff
@@ -0,0 +1,116 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Client
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            MainAsync(default).Wait();
+        }
```

Comment:
```suggestion
        {
            using var cts = new CancellationTokenSource();
            Console.CancelKeyPress += (o, e) => cts.Cancel();
            MainAsync(cts.Token).Wait();
        }
```

To demostrate working cancellation.